config/dpkg/python-plaso.install

```diff
@@ -1,6 +0,0 @@
-usr/lib/python2*/dist-packages/plaso/*.py
-usr/lib/python2*/dist-packages/plaso/*/*.py
-usr/lib/python2*/dist-packages/plaso/*/*.yaml
-usr/lib/python2*/dist-packages/plaso/*/*/*.py
-usr/lib/python2*/dist-packages/plaso/*/*/*.yaml
-usr/lib/python2*/dist-packages/plaso*.egg-info/*
```

Comment:
Is this just leftovers from PY2 times?

Reply:
Yes python-plaso.install was the packaging file for python-plaso deb